src/EloquentBuilder.php

```diff
@@ -2,13 +2,19 @@
 
 namespace DesignMyNight\Elasticsearch;
 
+use DesignMyNight\Elasticsearch\Contracts\FilterInterface;
 use Generator;
 use Illuminate\Database\Eloquent\Builder as BaseBuilder;
 use Illuminate\Database\Eloquent\Model;
 use Illuminate\Pagination\LengthAwarePaginator;
 use Illuminate\Pagination\Paginator;
 use Illuminate\Support\Collection;
 
+/**
+ * Class EloquentBuilder
+ * @method QueryBuilder filter(FilterInterface $filters)
```

Comment:
I can't find this filter method, where is it?

Reply:
It doesn't exist which is why I've added that @method annotation. Its so that an IDE will pick up the filter query scope method. I only added it in so PHPStorm would stop highlighting it as method not found.

```php
    /**
     * @param Builder         $query
     * @param FilterInterface $filters
     *
     * @return Builder
     */
    public function scopeFilter(Builder $query, FilterInterface $filters): Builder
    {
        return $filters->apply($query);
    }
```

`$this->query = Occurrence::newElasticsearchQuery()->filter($filters);`

Reply:
Ahh ok. Would it be clearer (and would it work) to have that hint on Model since `scopeFilter` is a method on Model rather than on EloquentBuilder?

Reply:
Ah, annoying - fine to leave it here then.